contrib/builder/deb/armhf/generate.sh

```diff
@@ -83,15 +83,15 @@ for version in "${versions[@]}"; do
 	)
 	# packaging for "sd-journal.h" and libraries varies
 	case "$suite" in
-		precise|wheezy) ;;
-		jessie|trusty) packages+=( libsystemd-journal-dev );;
-		*) packages+=( libsystemd-dev );;
+		wheezy) ;;
+		jessie|trusty) packages+=( libsystemd-journal-dev ) ;;
+		*) packages+=( libsystemd-dev ) ;;
```

Comment:
`;;` is part of the `case` construct, probably needs to be a distinct word so that it does not get confused with the command delimiter `;` (?)